examples/relationships/README.md

```diff
@@ -0,0 +1,209 @@
+
+Once you create the content models, you could create content items based on these and the site would be capable to handle the content and render it.
+
+### Kontent content models
+
+Kontent project contains a simple article content base, that could be tagged. There are two content types `Article` and `Tag`. Kontent project is using two languages `en-US` and `cs-CZ` to showcase [Language variant relationships](#Language-variant-relationships);
+
+#### Article content type
+
+This content type - `Article item` - has following structure:
+
+- Title - **Text** element
+- Content - **Text** element
+- Date - **Date & Title** element
+- Slug - **URL slug** element - auto-generated from title
+- Tags - **Linked items** element - content type limited to `Tag`
+
+This content type represents an article, first four elements (`Title`, `Content`, and `Date`) are meant to be used for some sample content that should be displayed on site. The `Content` element is set to simple text because of simplicity. Rich text element type possibilities are described in [Resolution example](../resolution#readme). The `Slug` element is used to route registration. `Tags` element is modeled as Linked items element type because of [Linked from relationship](#Linked-from-relationship) example.
+
+#### Tag content type
+
+- Title - **Text** element
+- Slug - **URL slug** element - auto-generated from title
+
+This content type represents a simple tag for an article. An article could contain more than one. This tag is modeled because of [Linked from relationship](#Linked-from-relationship) example.
+
+## Features
+
+Following features are described on simple real-life use cases. All of them is using [Gatsby schema customization](https://www.gatsbyjs.com/docs/schema-customization) to connect GraphQL nodes among each other to achieve the queries simplify and  to make the site efficient.
+
+Schema extension are being performed right in the web site source code (in [gatsby-node.js](./gatsby-node.js) file) using [`createSchemaCustomization`](https://www.gatsbyjs.com/docs/node-apis/#createSchemaCustomization) method.
+
+If you want to separate this logic, it is possible to create a [local Gatsby plugin](https://www.gatsbyjs.com/docs/creating-a-local-plugin/) and define these customization there, as it is showcased in the [Kontent Lumen starter](https://github.com/Kentico/gatsby-starter-kontent-lumen/tree/master/plugins/kontent-used-by-content-items), just use [`createResolvers`](https://www.gatsbyjs.com/docs/node-apis/#createResolvers) method for the customizations. If you want to re-use this code, you could [publish the plugin to the npm](https://www.gatsbyjs.com/contributing/submit-to-plugin-library/) and then re-use it.
+
+> In case you published your schema extension bound to Kontent GraphQL nodes feel free to raise an issue/pull request with plugin description and we are happy to mentions it here!
```

Comment:
```suggestion
> In case you published your schema extension bound to Kontent GraphQL nodes feel free to raise an issue/pull request with plugin description and we are happy to mention it here!
```